.github/workflows/ROMROM-IOS-TESTFLIGHT.yaml

```
@@ -263,17 +263,47 @@ jobs:
               pilot(
                 api_key: api_key,
                 ipa: ENV["IPA_PATH"],
                 changelog: ENV["RELEASE_NOTES"],
-                skip_waiting_for_build_processing: false,  # 완료까지 기다림
-                groups: ["alom"],  # 자동 배포
+                skip_waiting_for_build_processing: true,  # 빠르게 업로드만
                 distribute_external: false,
                 notify_external_testers: false,
                 uses_non_exempt_encryption: false  # 암호화 규정 자동 설정
               )
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)